src/Native/Unix/CMakeLists.txt

```diff
@@ -152,7 +152,7 @@ if (NOT WIN32)
         # Ensure that objcopy is present
         if(DEFINED ENV{CROSSCOMPILE})
             if(CMAKE_SYSTEM_PROCESSOR STREQUAL armv7l OR CMAKE_SYSTEM_PROCESSOR STREQUAL aarch64 OR CMAKE_SYSTEM_PROCESSOR STREQUAL i686)
-                find_program(OBJCOPY ${TOOLCHAIN}-objcopy)
+                find_program(OBJCOPY ${TOOLCHAIN_PREFIX}objcopy)
```

Comment:
That's interesting, TOOLCHAIN_PREFIX is set for cross compiling on CoreCLR: https://github.com/dotnet/coreclr/search?utf8=%E2%9C%93&q=TOOLCHAIN_PREFIX

TOOLCHAIN_PREFIX contains a fully qualified path. That's required because when cross-compiling for Android, the toolchain is in a seperate toolchain folder which is not in $PATH. (When cross compiling for Linux, the host OS's cross compile toolchain is used, and that tends to be on $PATH).

I guess we could set TOOLCHAIN to a full path, though - but not sure why the scripts differ between CoreCLR and CoreFX.